doc/Publishing.md

```diff
@@ -87,6 +87,24 @@ For example, if you picked `0.1.0` as a version then you should run:
 
 To confirm that the tagging was correctly done, check the repo's [releases](https://github.com/eclipse-theia/theia/releases) page and confirm the release just done is listed there.
 
+## Updating the release page on eclipse
+- Login to eclipse: https://projects.eclipse.org/projects/ecd.theia
+- On the right side panel, select Release / Create a new release
+    Create a new version ex: 1.2.0
```

Comment:
This step is not descriptive enough, we should mention to fill out the **date** and the **name**.
We should describe what the **name** should be in the documentation (ex: `1.2.0`)

Reply:
Done in next commit